src/components/Auth/SignIn.tsx

```diff
@@ -9,18 +9,21 @@ import Layout from '../Layout'
 import Input from '../Utils/Input'
 import {useAsync} from '../Utils/hooks'
 
-const SignIn = () => {
-  const {useVerifyUserSignInCredentials} = useAuth(null)
+const SignIn = (): React.ReactNode => {
+  const {useVerifyUserSignInCredentials} = useAuth()
   const [
     verificationFailed,
     checkUserCredentials,
   ] = useVerifyUserSignInCredentials()
   const {status, dispatch} = useAsync()
 
-  async function submitUserCredentials(e) {
+  async function submitUserCredentials(e: React.SyntheticEvent) {
     e.preventDefault()
     dispatch({type: 'pending'})
-    const {email, password} = e.target.elements
+    const {email, password} = e.target as typeof e.target & {
```

Comment:
you lost `.elements` ... is that on purpose?

Reply:
yes, couldn't get it typed. i check the functionality and it was still working.